src/main/java/mate/academy/bookstore/model/Book.java

```diff
@@ -5,15 +5,21 @@
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
+import jakarta.persistence.Table;
 import java.math.BigDecimal;
 import lombok.Getter;
 import lombok.Setter;
 import lombok.ToString;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
 
 @Entity
 @Getter
 @Setter
 @ToString
+@SQLDelete(sql = "UPDATE book SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted=false")
```

Comment:
where is deperacted, replace with SQLRestriction
uppercase true and false 
```suggestion
@SQLRestriction("is_deleted = FALSE")
```